sites/game-client/client.ejs

```diff
@@ -22,10 +22,33 @@ absolute URL.
   </head>
   <body>
     <div id="rootdiv" class="container-center">
-      <% inDiv.forEach((i) => { %>
-        <%- include(i); %>
-      <% }); %>
-      <canvas id="mapcanvas"></canvas>
+      <div id="staging-div">
+        <!--
+        This div is for stuff that's not yet being displayed.
+        It is currently semi-transparent for debugging purpose.
+        -->
+        <div id="map-render-overlay" class="container-center">
+          <canvas id="mapcanvas" width="640" height="640"></canvas>
```

Comment:
The size of canvas is adjusted in `misc.js`. Maybe shouldn't assign the value in html?

Reply:
Sorry, forgot to remove when debugging.